nextjs-13/src/repositories/newsletter.repository.test.ts

```diff
@@ -0,0 +1,24 @@
+import { dbClientMock } from '@test/database';
+import { newsletterFactory } from '@test/factories/newsletter.factory';
+
+import { createNewsletter } from './newsletter.repository';
+
+describe('Newsletter Respository', () => {
+  describe('createNewsletter', () => {
+    it('creates a newsletter', async () => {
+      const newsletterAttributes = {
```

Comment:
Why you define an attribute here, actually just mock the data should be enough right ?

Reply:
Yes I don't need the `newsletterAttributes` for the mocking, but I need it for the [`createNewsletter` method](https://github.com/nimblehq/js-framework-benchmark/pull/96/files#diff-7f62d1d6ff486da9e5be846b0de3dc572e91704e2e913d9aa47ab9e57090f37bR4), which only accepts `Newsletter`, means that it only 3 fields `name`, `content`, and `user`.

Ah actually my current `newsletterAttributes` now is not correct, it should have the field `user` instead of `userId`. Let me fix this.

Reply:
Fixed: https://github.com/nimblehq/js-framework-benchmark/pull/96/commits